src/components/Categories.js

```diff
@@ -0,0 +1,12 @@
+import Navbar from './Navbar';
+
+function Categories() {
+  return (
+    <div>
+      <Navbar />
+      <button type="button">Categories</button>
```

Comment:
- [ ] Good job on implementing the button however the text should be `Check status` instead of `Categories`, kindly consider modifying the button text so it's meaningful

Reply:
@Ismailco Thank you so much for your suggestions.